### PR TITLE
bump version to 7.5.3

### PR DESCRIPTION
**Pull request by sebastienlagarde**
all in titlle

### CODE REVIEW

File: com.unity.render-pipelines.core/CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,11 @@ All notable changes to this package will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [7.5.3] - 2021-11-1
```
Comment (theo-pnv) [inline com.unity.render-pipelines.core/CHANGELOG.md:7]: iirc we had some validation tests issues in the past with dates with only one digit (like this one).
Please add a `0` to the date: `2021-11-01`

Comment (theo-pnv) [reply com.unity.render-pipelines.core/CHANGELOG.md:7]: Note: For all packages

**Review by theopnv — CHANGES_REQUESTED** [review]
Do lines 4 and 9 of the visual effect graph testing package need to be bumped as well? https://github.com/Unity-Technologies/Graphics/blob/srp-7-5-3/com.unity.testing.visualeffectgraph/package.json#L4

File: com.unity.render-pipelines.core/CHANGELOG.md (hunk 1 above)
Comment (phi-lira) [reply com.unity.render-pipelines.core/CHANGELOG.md:7]: looking at the history, shouldn't the date/month be reversed as well? 
`2021-01-11`

**Review by phi-lira — CHANGES_REQUESTED** [review]

File: com.unity.render-pipelines.core/CHANGELOG.md (hunk 1 above)
Comment (sebastienlagarde) [reply com.unity.render-pipelines.core/CHANGELOG.md:7]: good point!

**Comment by sebastienlagarde** [comment]
adressed pr feedback

**Review by theopnv — APPROVED** [review]

**Comment by sebastienlagarde** [comment]
thanks for the update of testing vfx package

**Pull request merged**